api/v1beta3/authpolicy_types.go

```diff
@@ -1,4 +1,4 @@
-package v1beta2
+package v1beta3
```

Comment:
I think we can also remove all `+kubebuilder:validation:MaxProperties=10` markers. They were added to keep the CRD within the complexity budget when we first introduced reoute selectors.

Reply:
Yap can update this. I expect this will should also effect some tests also.